common/src/main/java/com/wynntils/screens/chattabs/widgets/ChatTabSettingsButton.java

```diff
@@ -33,13 +33,13 @@ public void renderWidget(GuiGraphics guiGraphics, int mouseX, int mouseY, float
         FontRenderer.getInstance()
                 .renderAlignedTextInBox(
                         poseStack,
-                        StyledText.fromString("+"),
+                        StyledText.fromString("\uD83D\uDDD7"), // Tabs Icon
```

Comment:
Note for other devs, should we have a preference on writing unicode like this vs the actual character? If you copy the character from somewhere like https://www.compart.com/en/unicode/ and paste as plain text into Intellij it works fine. I try to use the actual character as it's more readable imo but I know we do it like this in a lot of other places

Reply:
Iirc, general standard is to type it out whenever it is displayed, etc, and use unicode when used in patterns/string matching.

Reply:
We tried copying the actual character here, Intellij just kept giving us the code though.

Reply:
I can get it using this if I copy the actual character
![image](https://github.com/Wynntils/Artemis/assets/8337467/63be6dd0-c7c9-4373-8ecd-29a9ee095e8d)